pycontrails/models/cocip/cocip.py

```diff
@@ -712,8 +712,9 @@ def _simulate_wake_vortex_downwash(self, met: MetDataset) -> None:
         wingspan = self._sac_flight.get_data_or_attr("wingspan")
 
         # get the pressure level `dz_m` lower than element pressure
-        air_pressure_lower = thermo.p_dz(air_temperature, air_pressure, self.params["dz_m"])
-        level_lower = air_pressure_lower / 100
+        dz_m = self.params["dz_m"]
+        air_pressure_lower = thermo.p_dz(air_temperature, air_pressure, dz_m)
```

Comment:
we should really rename `thermo.p_dz` to something more expressive, even `pressure_dz` would be better.

Reply:
cbb961f1774b3081875f7e24f